client/src/components/Home.js

```diff
@@ -1,30 +1,98 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
 import Icon from '@material-ui/core/Icon';
 import Grid from '@material-ui/core/Grid';
 import { withStyles } from '@material-ui/core/styles';
 import Leaderboard from './Leaderboard';
+import LoaderComponent from './common/loaderComponent/loaderComponent';
```

Comment:
capitalize component filename

Reply:
Will refactor after Prakhar branch is merged because he has taken a branch out of this branch.

Reply:
@vkbinfo You can go ahead and refactor. I can always rebase and resolve conflicts.